app/components/collaboration.tsx

```diff
@@ -0,0 +1,49 @@
+import { MoveRight } from "lucide-react";
+import { Link } from "@remix-run/react";
+
+import indonesiaMapSvg from "~/assets/map.svg";
+import indonesiaMap from "~/assets/map.png";
+import { Button } from "./ui/button";
+
+export default function Collaboration() {
+  return (
+    <>
+      <div className="flex gap-2 justify-center items-end mb-10">
+        <div className="xs:text-xl lg:text-6xl font-semibold ">Kolaborasi</div>
+        <div className="xs:text-base lg:text-2xl font-medium text-[#c7cbd1]">
+          dengan
+        </div>
+        <img
+          className="w-[67.44px] h-[31.57px]"
+          src="./images/logos/reactjsid.svg"
+          alt=""
+        />
+      </div>
+      <div className="font-medium xs:text-base  lg:text-2xl mb-12 xs:px-2 lg:px-10 text-[#c7cbd1]">
+        Kami di reactjs.id menyelenggarakan event untuk developer React JS
+        indonesia berbagi wawasan dan pengetahuan mengenai ekosistem Reactjs.
+      </div>
+
+      <div className="relative">
+        <img src={indonesiaMap} alt="" className="w-full h-full" />
+      </div>
+      <div className=" bg-black relative">
+        <img src={indonesiaMapSvg} alt="" className="w-full h-full" />
```

Comment:
just curious, why `indonesiaMap` and `indonesiaMapSvg` put on `assets` folder and using `import`? why not treat that same as logo on above (`"./images/logos/reactjsid.svg"`)?

Reply:
open to discuss. i assume /public folder will be uploaded to s3 which mean rarely changed such: favicon is ok to put there. also /public is more general can reused by another page / component. with that, i prefer put the map asset in local because it's specific to usecase. CMIIW :)